addons/mail/static/tests/chat_bubble/chat_bubble.test.js

```diff
@@ -353,6 +355,13 @@ test("Can close all chat windows at once", async () => {
     await click("button.fa.fa-ellipsis-h[aria-label='Chat Hub Options']");
     await click("button.o-mail-ChatHub-option", { text: "Close all conversations" });
     await contains(".o-mail-ChatBubble", { count: 0 });
+    const members = pyEnv["discuss.channel.member"].search_read([
```

Comment:
We should add more guarantee the RPC was done (at least `onRpc`), trigger on rendering is not sufficient before checking server data.
(with onRpc we probably don't need subsequent data check, just assert steps)

Reply:
I used `onRpcBefore` and added a way to await `"closed"` of all `channel/fold` (the 20 channels).